public/docs/ts/latest/cli-quickstart.jade

```diff
@@ -122,157 +110,263 @@ figure.image-display
-  Notice that we import the `bootstrap` function from `@angular/platform-browser-dynamic`,
-  not `@angular/core`.
-  Bootstrapping isn't core because there isn't a single way to bootstrap the app.
-  True, most applications that run in a browser call the bootstrap function from
-  this library.
-
-  But it is possible to load a component in a different environment.
-  We might load it on a mobile device with [Apache Cordova](https://cordova.apache.org/) or [NativeScript](https://www.nativescript.org/).
-  We might wish to render the first page of our application on the server
-  to improve launch performance or facilitate
-  [SEO](http://www.google.com/webmasters/docs/search-engine-optimization-starter-guide.pdf).
-  These targets require a different kind of bootstrap function that we'd import from a different library.
-
-  ### Why create a *main.ts* file?
-
-  Both `main.ts` and the application component files are tiny.
-  This is just a QuickStart.
-  We could have merged these two files into one
-  and spared ourselves some complexity.
-
-  We'd rather demonstrate the proper way to structure an Angular application.
-  App bootstrapping is a separate concern from presenting a view.
-  Mixing concerns creates difficulties down the road.
-  We might launch the `CliQuickstartAppComponent` in multiple environments with different bootstrappers.
-  Testing the component is much easier if it doesn't also try to run the entire application.
-  Let's make the small extra effort to do it *the right way*.
-  
-  ### Loading the application with SystemJS
-  
-  The CLI uses `System.js` to load the application. We just need to call `System.import` and pass it our `main.ts`
-  file to boot our application.
-  
-+makeExcerpt('src/index.html', 'import')
+block src-files
+  :marked
+    ### The `src` folder
+    Your app lives in the `src` folder.
+    All Angular components, templates, styles, images and anything else your app needs goes in here.
+    Any files outside of this folder are meant to support building your app.
+
+.filetree
+  .file src
+  .children
+    .file app
+    .children
+      .file app.component.css
+      .file app.component.html
+      .file app.component.spec.ts
+      .file app.component.ts
+      .file app.module.ts
+    .file assets
+    .children
+      .file .gitkeep
+    .file environments
+    .children
+      .file environment.prod.ts
+      .file environment.ts
+    .file favicon.ico
+    .file index.html
+    .file main.ts
+    .file polyfills.ts
+    .file styles.css
+    .file test.ts
+    .file tsconfig.json
+
+style td, th {vertical-align: top}
+table(width="100%")
+  col(width="20%")
+  col(width="80%")
+  tr
+    th File
+    th Purpose
+  tr
+    td <code>app/app.component.{ts,html,css,spec.ts}</code>
+    td
+      :marked
+        Defines the `AppComponent` along with an HTML template, CSS stylesheet and a unit test.
+        It is the **root** component of what will become a tree of nested components
```

Comment:
Are we still calling this component a root component? Or was bootstrap component? This is a honest doubt :P